react/src/components/carousel/TextboxContent.js

```diff
@@ -0,0 +1,18 @@
+import TextboxItem from './TextboxItem';
+import Styles from './TextboxContent.module.css';
+
+const loremIpsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum ac neque bibendum, vehicula ipsum et, tincidunt ligula.";
+const content = [];
+for(let i = 1; i <=6; i++) {
+    content.push(<TextboxItem itemNumber={i} itemText={loremIpsum} key={i}/>);
+}
```

Comment:
Do contentu wykorzystajmy `useState`. `for` i `.push` dajmy w `useEffect` z pustą tablicą (`useEffect(() => {}, [])`) - wtedy useEffect wykona się tylko raz.